### PR TITLE
Add gift card support

### PR DESCRIPTION
**Pull request by stephtaylor**

### CODE REVIEW

File: docs/giftcards.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+id: giftcards
+title: Accepting Gift Cards
+---
+Accept gift cards using the Square Payment Form.
+
+https://developer.squareup.com/docs/payment-form/gift-cards/part-1
+
+> **Important**: If you are adding Gift Cards to a payment page that already uses an instance of the Square Payment Form for credit cards, you **must** initialize a second Square Payment Form for the gift cards. How to [Render Multiple Payment Forms](customization.md#render-multiple-payment-forms)
+
+Gift cards are *not* supported in Sandbox at this time. See the docs for more information.
```
Comment (maxbeatty) [inline docs/giftcards.md:11]: Square Gift Card testing value was adding relatively recently https://developer.squareup.com/docs/testing/test-values#card-present-success-state-values

Comment (stephtaylor) [reply docs/giftcards.md:11]: Oh awesome! We need to update this doc then if it's not already in progress: https://developer.squareup.com/docs/payment-form/gift-cards/part-1#before-you-start

**Comment by spenweb** [comment]
When should we expect a release to NPM with this new gift card improvement? The documentation website leads me to think that it the latest from NPM should already have gift cards, but this has not been released yet.

**Comment by stephtaylor** [comment]
@spenweb Thank you for reminding us to actually publish the changes 🤦  v0.7.0 should have the gift card changes.

Let us know if you run into issues or have any suggestions!

**Pull request merged**